projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/main.yml

```diff
@@ -117,6 +140,22 @@
     state: directory
     mode: '0755'
 
+- name: Set s3_host_base
+  block:
+  - name: Get the route hostname
+    command: 
+      oc get route/minio-secure -o jsonpath={.spec.host} -n {{ notebooks_dashboard_scale_test_minio_namespace }}
+    register: hostname_result
+
+  - name: Set s3_host_base fact
+    set_fact:
+      s3_host_base: "{{ hostname_result.stdout }}"
+
+  - name: Debug the s3_host_base variable (optional)
+    debug:
+      msg: "s3_host_base is set to {{ s3_host_base }}"
+
+
```

Comment:
you can remove these changes. The hostname is visible in the logs of the command above